src/test/scala/com/mozilla/telemetry/streaming/TestEventsToAmplitude.scala

```diff
@@ -171,13 +219,160 @@ class TestEventsToAmplitude extends FlatSpec with Matchers with BeforeAndAfterAl
       "--kafkaBroker", Kafka.kafkaBrokers,
       "--startingOffsets", "latest",
       "--url", s"http://$Host:$Port$path",
-      "--config-file-path", configFilePath,
+      "--config-file-path", configFilePath(ConfigFileName),
       "--raise-on-error")
     val opts = new EventsToAmplitude.Opts(args)
 
     EventsToAmplitude.process(opts, apiKey)
 
     kafkaProducer.close
     spark.streams.removeListener(listener)
+
+    verify(expectedTotalMsgs, createMatcher(focusEventJsonMatch))
   }
+
+  "Events to Amplitude" should "send main ping events via HTTP request" taggedAs(Kafka.DockerComposeTag, DockerEventsTag) in {
```

Comment:
I would probably change the `DockerEventsTag` to a different one. Then we can use the `DockerComposeTag` to run or exclude the docker tests, and run specifically the main events to Amplitude tests using the new tag (unless you see a specific reason to lump the main events to Amplitude and the focus-event events to Amplitude jobs together).

Reply:
I mean, to be clear, I just copied-and-pasted without thinking about it too much but I do think one consideration is that since they share so much code it's definitely easy to affect one while working on the other, so it seems like it'd make sense to exercise both on these test runs.

Reply:
Maybe then we can have one tag for all `Amplitude` related stuff, and a subtag for the Main Ping and Focus-Event.